src/tools/rest-backend/authentication_application.cpp

```diff
@@ -316,4 +304,46 @@ model::User AuthenticationApp::getCurrentUser (cppcms::http::request & request)
 	}
 }
 
+/**
+ * @brief can be used to build a jwt for a user
+ *
+ * @param user a user model
+ * @return a jwt token as string
+ */
+std::string AuthenticationApp::buildJWT (cppcms::http::response & resp, const model::User & user) const
```

Comment:
I actually had this improvement already in mind. Let's make a separate ticket for possible improvements for the future, if I get bored. ;)